requirements/edx/base.txt

```diff
@@ -87,6 +87,12 @@ django-ratelimit-backend==0.6
 unicodecsv==0.9.4
 django-require==1.0.6
 
+# Used for shopping cart's pdf invoice/receipt generation
+reportlab==3.1.44
+
+# Used for extracting/parsing pdf text
+pdfminer==20140328
```

Comment:
@nedbat I presume you are the right person to sign off regarding license compatibility with these two libraries.